src/Ocis.php

```diff
@@ -57,20 +60,24 @@ public function __construct(
         $this->accessToken = $accessToken;
         $this->guzzle = new Client(self::createGuzzleConfig($connectionConfig, $this->accessToken));
         if (!self::isConnectionConfigValid($connectionConfig)) {
-            throw new \InvalidArgumentException('connection configuration not valid');
+            throw new \InvalidArgumentException('Connection configuration is not valid');
         }
         $this->connectionConfig = $connectionConfig;
         $this->graphApiConfig = Configuration::getDefaultConfiguration()->setHost($serviceUrl . '/graph/v1.0');
 
     }
 
+    /**
+     * @ignore This function is mainly for unit tests and should not be shown in the documentation
+     */
     public function setGuzzle(Client $guzzle): void
     {
         $this->guzzle = $guzzle;
     }
 
     /**
      * @param array<mixed> $connectionConfig
+     * @ignore This function is used for internal purposes only and should not be shown in the documentation. The function is public to make it testable and because its used also from other classes.
```

Comment:
```suggestion
     * @ignore This function is used for internal purposes only and should not be shown in the documentation. The function is public to make it testable and because its also used from other classes.
```